projects/igniteui-angular/src/lib/test-utils/calendar-helper-utils.ts

```diff
@@ -64,9 +64,9 @@ export class HelperTestFunctions {
         const date = fixture.nativeElement.querySelector(HelperTestFunctions.CALENDAR_HEADER_DATE_CSSCLASS);
         expect(date).not.toBeNull();
 
-        const dateParts = selectedDate.toUTCString().split(' '); // (weekday, date month year)
```

Comment:
toUTCString() was shifting time changing the day
the corrected version keeps everything in local time